Sources/CreateOptions/PackageDeclaration.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+public enum DependencyRule: Decodable {
+    case exact(version: String)
+    case upToNextMajor(from: String)
+    case upToNextMinor(from: String)
```

Comment:
I propose that we drop these since they are deprecated in Swift 5.6 with the intention that you specify the range yourself. Once our Package.swift targets 5.6 above, we'd have to manually deconstruct the string and figure it out ourselves ([like this](https://github.com/apple/swift-package-manager/blob/8970e016e9688f0e1fca549cdc02719fa5359059/Sources/PackageDescription/PackageRequirement.swift#L216-L231)) which isn't worth it. 

If we don't include it now, it'll encourage people to use `range` or `from` directly which will translate into a Swift 5.6 friendly format once that item comes (probably after 5.7 is publicly released so in two months)